creator-node/src/apiSigning.ts

```diff
@@ -183,18 +189,18 @@ const verifyRequesterIsValidSP = async ({
  * @param {string} spID
  * @returns a parsed spID
  */
-function validateSPId(spID) {
+function validateSPId(spID: string): number {
   if (!spID) {
     throw new Error('Must provide all required query parameters: spID')
   }
 
-  spID = parseInt(spID)
+  const spIDNumber = parseInt(spID)
 
-  if (isNaN(spID) || spID < 0) {
-    throw new Error(`Provided spID is not a valid id. spID=${spID}`)
+  if (isNaN(spIDNumber) || spIDNumber < 0) {
+    throw new Error(`Provided spID is not a valid id. spID=${spIDNumber}`)
   }
 
-  return spID
+  return spIDNumber
 }
 
 module.exports = {
```

Comment:
Are you able to get rid of the `module.exports` at the bottom and export the members? It can be weird sometimes if the file is imported by a bunch of `js` but I think this one is safe

Reply:
yes! done